include/ttmlir/Target/Common/types.fbs

```diff
@@ -224,8 +224,8 @@ table EventRef {
 }
 
 enum MathFidelity : uint8 {
-    LoFi          = 0,
-    HiFi2         = 2,
-    HiFi3         = 3,
-    HiFi4         = 4,
+    LoFi,
+    HiFi2,
+    HiFi3,
+    HiFi4,
 }
```

Comment:
This is the only case where enum is not sequental -- value `1` is skipped. I think this was done just to match `HiFi` naming, no other reason.

If I am mistaken, please correct me.

Reply:
In some cases we do cast from the flatbuffer type to the metal type, I think we have static_asserts in all of the places that rely on this behavior. 🤞

Reply:
Maybe better to leave explicit values in enums like this one where values are not sequential? And to put comment above enum documenting why it was done. To avoid potential hidden issues where someone relied on enum values to match the metal ones.